docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -165,3 +162,34 @@ will be lost.
 ```bash
 kubectl delete -f alluxio-journal-volume.yaml
 ```
+
+## Advanced Setup
+
+### POSIX API
+
+Once Alluxio is deployed on Kubernetes, there are multiple ways in which a client application can
+connect to it. For applications using the [POSIX API]({{ '/en/api/POSIX-API.html' | relativize_url }}),
+application containers can simply mount the Alluxio FileSystem.
+
+In order to use the POSIX API, first deploy the Alluxio FUSE daemon.
+```bash
+cp alluxio-fuse.yaml.template alluxio-fuse.yaml
+kubectl create -f alluxio-fuse.yaml
+```
+Note:
+- The container running the Alluxio FUSE daemon must have the `securityContext.privileged=true` with
+SYS_ADMIN capabilities. Application containers that require to access Alluxio do not require this
+privilege.
+- A different docker image [alluxio/alluxio-fuse](https://hub.docker.com/r/alluxio/alluxio-fuse/) based
+on `ubuntu` instead of `alpine` is needed to run the FUSE daemon. Application containers can run on
+any docker image.
+
+Verify that a container can simply mount the Alluxio FileSystem without any custom binaries or
+capabilities using a `hostPath` mount of location `/alluxio-fuse`:
+```bash
+cp alluxio-fuse-client.yaml.template alluxio-fuse-client.yaml
+kubectl create -f alluxio-fuse-client.yaml
+```
+
+If using the template, Alluxio is mounted at `/alluxio-fuse` and can be accessed via the POSIX-API
+across multiple containers.
```

Comment:
Could we add an instruction to show users how to get the alluxio fuse logs? Since fuse console output is sometimes misleading and fuse logs is important to our users.

Reply:
@LuQQiu when you say fuse logs are you talking about this output `/opt/alluxio/logs/fuse.log` or some other logs?

Reply:
Yes! the fuse.log

Reply:
Thanks @LuQQiu. Added a troubleshooting section below